docs/lib/datastore/fragments/flutter/getting-started/70_querySnippet.md

```diff
@@ -1,3 +1,7 @@
 ```dart
-TBD
-```
\ No newline at end of file
+try {
+  List<Post> posts = query(Post.classType);
```

Comment:
Does it not need to be:
```suggestion
  List<Post> posts = await Amplify.DataStore.query(Post.classType);
```

Reply:
You are right, bad copy paste error. Thanks for catching.